libs/features/webapp-prompt/tooltip.css

```diff
@@ -0,0 +1,50 @@
+[data-pep-dismissal-tooltip]::after {
+  content: attr(data-pep-dismissal-tooltip);
+  position: absolute;
+  display: inline-flex;
+  z-index: 3;
+  background-color: #3B63FB;
+  height: fit-content;
+  width: 8.875rem;
+  top: 120%; 
+  right: -75%;
+  word-break: break-word;
+  border-radius: 7px;
+
+  padding-inline-start: 0.5625rem;
+  padding-inline-end: 0.5625rem;
+  padding-block-start: 0.25rem;
+  padding-block-end: 0.3125rem;
+  
+  transition: opacity 0.5s;
+
+  font-family: 'Adobe Clean';
+  font-size: 0.75rem;
+  line-height: 0.9375rem;
+  color: white;
+  pointer-events: none;
+}
+
+[data-pep-dismissal-tooltip]::before { 
+  content: '';
+  position: absolute;
+  z-index: 2;
+  background-color: #3B63FB;
+  width: 0.44rem;
+  height: 0.44rem;
+  border-radius: 0.05rem;
+  left: calc(50% - 0.22rem);
+  top: 110%;
+  pointer-events: none;
+  
+  transform: rotate(45deg);
+
+  transition: opacity 0.5s;
+}
+
+@media (min-width: 1520px) {
+  [data-pep-dismissal-tooltip]::after {
+    left: calc(50% - 5rem);
+    right: 0;
```

Comment:
Same RTL question

Reply:
see RTL comment below. I've also redone the absolute positioning to only use `top` and `left`